book/scripts.md

```diff
@@ -103,7 +103,7 @@ Hello string +1
 
 ## Subcommands
 
-A script can have multiple sub-commands like `run`, `build`, etc. which allows to execute a specific main sub-function. The important part is to expose them correctly with `def main [] {}`. See more details in the [Custom Command](custom_commands.html#sub-commands) section.
+A script's `main` command can have multiple [subcommands](custom_commands.html#subcommands) like `run`, `build`, etc. which allows executing a specific sub-function from the commandline.
```

Comment:
```suggestion
A script can have multiple [subcommands](custom_commands.html#subcommands) like `run` or `build`.
```

To the user of the script, it's a subcommand of the script, not of `main`.  I think the original wording is probably better here, but still like your other change here.